charts/fint-oneroster-rest-provider/templates/service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "fint-oneroster-rest-provider.name" . }}
+  annotations:
+    service.beta.kubernetes.io/azure-load-balancer-internal: "true"
```

Comment:
possible use helmet on this value aswell? 
e.g
metadata:
      {{- with .Values.service.annotations }}
      annotations:
        {{- toYaml . | nindent 8 }}
      {{- end }}

Reply:
changed to use values instead